addons/mediarecorder/userscript.js

```diff
@@ -298,7 +316,21 @@ export default async ({ addon, console, msg }) => {
         stopSignFunc = () => stopRecording();
         vm.runtime.once("PROJECT_STOP_ALL", stopSignFunc);
       }
-      recorder.start(1000);
+
+      // Delay
+      const delay = opts.delay || 0;
+      const roundedDelay = Math.floor(delay);
+      for (let index = 0; index < roundedDelay; index++) {
+        console.log(roundedDelay - index);
+        recordElem.textContent = msg("starting-in", { secs: roundedDelay - index });
+        await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Maybe waiting 950ms would be more accurate overall.

Reply:
Why?

Reply:
Browsers rarely take exactly the amount of milliseconds you specify, it's often more than that